### ISSUE TITLE
Compatible board, BOM building

### ISSUE THREAD

**Issue by laptopology** [techdregs/Home-Assitant-Voice-Remote#3]
Hello,
Thanks for creating this awesome project!
sorry if this is asked before but,
can I use WEMOS Lite V1.0.0 ESP32 ?
It can be found here: [aliexpress](https://www.aliexpress.com/item/4000038780903.html)
Also, searching for the speaker, looks like it's for Nintendo NS ...true?
_And last, what about the specs for the battery? (dimensions etc)_ ...oops, just saw, it's a 502248 lipo

**Comment by techdregs** [comment]
That board will not be a drop in replacement for the Feather v2, but the base voice assistant code will work with any esp32 board if it's modified correctly. The LED stuff won't work without appropriate hardware either.

I'm not sure about the speaker. It's just a 20x14mm speaker from Amazon.

I think this is the battery I used: https://www.amazon.com/battery-Rechargeable-Lithium-Polymer-Connector/dp/B07BTKPSNP/

**Comment by laptopology** [comment]
I guess everyone's concern is all about on-battery duration or time between charges with regular use. 
I never had a "sleeping" device 'couse I plug them to the nearest 5v adapter and forget about, so I don't have any experience with deep-sleep function.
But this is a Remote!
I think/know that ESP32 chip is a power hungry on any board (they get hot too!)
So, the real question is: from your experience/knowledge the wemos board can replace the invaluable features of the feather in terms of charging and deep sleeping? 
In terms of size I found (searching here and there) that the wemos is a bit smaller (also cheaper and easier to find).
And as for the led, we can use an external rgb (cut from a strip). Hey! we can use 2 leds! they're so cheap!

**Comment by techdregs** [comment]
Deep sleep battery usage in my setup is excellent. I haven't had to recharge my remote yet, but it is also almost never used, so this is just the once a day wake and deep sleep usage. See: https://youtu.be/4EZYYPmQnJY

Different boards will have different power draws due to the voltage regulators they use. I have no experience with that board, so I can't comment, but it appears to have the required circuitry to use a battery. That said, I have no idea what that board's deep sleep current draw is, and I've seen boards with mA of deep sleep current vs under 50uA for the board I chose. I believe on-state power usage should be similar across boards unless they have unique features, as this will be dominated by the wifi power demands of the ESP32. All ESP32 boards support deep sleep, it's just a matter of what else they put on the board that draws power when the main module is sleeping.

**Comment by laptopology** [comment]
Very informative video!
I guess I'll find out about my board of choice ;)
The cost for everything will be about 10-15 bucks, what can go wrong!
Code changes will all about GPIO numbers and the -external- led.
The enclosure will need some modification though (I don't think I can manage that part)

**Comment by techdregs** [comment]
Yeah code change shouldn't be complicated. I'd see if the board has an internal voltage sense pin for the battery to monitor levels. If not, you'd also need to use an analog pin and voltage divider to monitor battery level.
